src/Domain/Insights/ForbiddenDefineGlobalConstants.php

```diff
@@ -31,6 +28,9 @@ public function getDetails(): array
         $ignore = $this->config['ignore'] ?? [];
 
         $globalConstants = array_diff($this->collector->getGlobalConstants(), $ignore);
+        $globalConstants = array_filter($globalConstants, function ($key) {
```

Comment:
```suggestion
        $globalConstants = array_filter($globalConstants, static function ($key): bool {
```

You have duplication on this method actually. Not sure if we can remove that or not.

Reply:
I moved it into abstract Insight class :wink: